dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -143,6 +158,24 @@ public String getEndpointsInfo() {
 
         return msg.toString();
     }
+    
+    private <T> boolean containsAny(Collection<T> collection, Collection<T> values) {
+        if (values.size() == 0) {
+            return false;
+        }
+    
+        for (T value : values) {
+            if (collection.contains(value)) {
+                return true;
+            }
+        }
+        
+        return false;
+    }
+    
+    private String cleanUpPath(String path) {
+        return path.replaceAll("//+", "/");
```

Comment:
Also, could you add a unit test which verifies that path cleaning works correctly?